packages/flutter/lib/src/material/drawer.dart

```diff
@@ -190,6 +192,19 @@ class DrawerController extends StatefulWidget {
   /// Optional callback that is called when a [Drawer] is opened or closed.
   final DrawerCallback drawerCallback;
 
+  /// {@template flutter.material.drawer.dragStartBehavior}
+  /// Determines the way that drag start behavior is handled.
```

Comment:
It's a little strange that you have so many templates that basically describe the same thing. Could you just have one general template by restructuring the text a little? At each place where you use the template you could have one very specific sentence (e.g. this controls the drag behavior for opening the drawer) followed by the inserted template that describes the difference in drag behavior in general terms?

Reply:
That's what I originally had, but Ian wanted me to be specific about how the drag behavior affects each widget. Do you feel like that specification isn't necessary?

Reply:
No, I do feel the specification is necessary. I would just split it up in two: have one or two very specific sentences on each property that describe specifically what drag behavior is modified by this property (e.g. the drag gesture to ope the drawer). Then follow that up by inserting a generic template that describes the difference between the two behaviors.